### PR TITLE
Pattern Lab skeleton

### PR DESCRIPTION
**Pull request by bshyong**
Adding some elements and colors and getting patternlab up and running

@crwallace I could figure out an easy way to pull in all of our VA stylesheets so I took the main stylesheet from the production site—not optimal but I think it will work for now...

screen shot 2016-09-28 at 1 57 59 pm

### CODE REVIEW

**Comment by crwallace** [comment]
It looks like we have some duplicate files in `source/`, should these be there?

**Comment by bshyong** [comment]
@crwallace which files are duplicated?  The `_patterns` folder is primarily the place where we need to make modifications—some of the other stuff is related to the pattern lab library and how it builds the static files

**Comment by bshyong** [comment]
@crwallace Added a couple more elements like form fields and process lists. Let's review/regroup on this before proceeding further. I'd like to know how helpful these patterns are and whether it's meeting the initial goals for this project.

screen shot 2016-09-30 at 12 06 33 pm

---

screen shot 2016-09-30 at 12 59 35 pm

**Comment by crwallace** [comment]
@bshyong -- Is `Gruntfile.js`, `LICENSE`, `package.json`, and `patternlab-config.json` in `source/` the same as `Gruntfile.js`, `LICENSE`, `package.json`, and `patternlab-config.json` in the root of the directory? If so, do we need both of them?

File: patternlab-readme.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
+# Pattern Lab Node - Grunt Edition
```
Comment (bshyong) [inline patternlab-readme.md]: Good idea — done!

File: .gitignore [hunk 2]
```diff
@@ -35,3 +35,10 @@ jspm_packages
 
 # Optional REPL history
 .node_repl_history
+
+.DS_Store
+patternlab.json
```
Comment (bshyong) [inline .gitignore]: I'm not sure what that is but `patternlab.json` doesn't exist so I removed that line from `.gitignore`

**Comment by bshyong** [comment]
Just addressed feedback in comments above via commit    e51c225

**Review by crwallace — APPROVED** [review]

**Pull request merged**